nucleus/map_label/MapLabel.cpp

```diff
@@ -0,0 +1,108 @@
+/*****************************************************************************
+ * Alpine Terrain Renderer
+ * Copyright (C) 2024 Lucas Dworschak
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *****************************************************************************/
+
+#include "MapLabel.h"
+
+#include "CharUtils.h"
+#include "srs.h"
+
+namespace nucleus {
+
+void MapLabel::init(const std::unordered_map<int, const MapLabel::CharData>& character_data, const stbtt_fontinfo* fontinfo)
+{
+    float offset_x = 0;
+    float offset_y = -font_size / 2.0f + 75.0;
```

Comment:
use const or constexpr whenever possible (this gives additional optimisation hints for the compiler)